<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolCircle</class>
 <widget class="QFrame" name="ToolCircle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ToolCircle
{
	background-color: rgba(62, 62, 62, 60%);
	border-radius:20px;
}
</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stkWidget_Main">
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="toolPage1">
      <property name="styleSheet">
       <string notr="true">QToolButton
{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.711864 rgba(255, 255, 255, 255), stop:1 rgba(180, 180, 180, 255));

border-radius:35px;
}

QToolButton:pressed
{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.711864 rgba(170, 170, 170, 255), stop:1 rgba(100, 100, 100, 255));
}</string>
      </property>
      <widget class="SizeTBtn" name="tbtn_Tool">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/logo_btn_Tool_Off.png</normaloff>:/pic/logo_btn_Tool_Off.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="SizeTBtn" name="tbtn_Set">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/btn_Options_Enable.png</normaloff>:/pic/btn_Options_Enable.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="SizeTBtn" name="tbtn_Home">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>主页</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/black_Home_Ori.png</normaloff>:/pic/black_Home_Ori.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="SizeTBtn" name="tbtn_Return">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/btn_Undo2_off.png</normaloff>:/pic/btn_Undo2_off.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="SizeTBtn" name="tbtn_KeyBoard">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.711864 rgba(255, 255, 255, 255), stop:1 rgba(180, 180, 180, 255));

border-radius:35px;
}

QToolButton:pressed
{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.711864 rgba(170, 170, 170, 255), stop:1 rgba(100, 100, 100, 255));
}</string>
       </property>
       <property name="text">
        <string>键盘</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/keyboard_off.png</normaloff>:/pic/keyboard_off.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="toolPage2">
      <property name="styleSheet">
       <string notr="true">QToolButton
{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.711864 rgba(255, 255, 255, 255), stop:1 rgba(180, 180, 180, 255));

border-radius:35px;
}

QToolButton:pressed
{
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.711864 rgba(170, 170, 170, 255), stop:1 rgba(100, 100, 100, 255));
}</string>
      </property>
      <widget class="SizeTBtn" name="tbtn_PrintScreen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>截图</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/screenshot.png</normaloff>:/pic/screenshot.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="SizeTBtn" name="tbtn_Return_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/btn_Undo2_off.png</normaloff>:/pic/btn_Undo2_off.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="SizeTBtn" name="tbtn_Sys">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>系统</string>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="SizeTBtn" name="tbtn_Help">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>70</width>
         <height>70</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>系统</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/pic/help.png</normaloff>:/pic/help.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SizeTBtn</class>
   <extends>QToolButton</extends>
   <header location="global">./SpecialUI/sizetbtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
